soundconverter.appdata.xml.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/data/soundconverter.appdata.xml b/data/soundconverter.appdata.xml
```

Comment:
Should be upstreamed.

Reply:
See https://github.com/kassoulet/soundconverter/pull/46